Bump blebox_uniapi to 2.2.1

swistakm opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).
-->

## Proposed change
<!--
  Describe the big picture of your changes here to communicate to the
  maintainers why we should accept this pull request. If it fixes a bug
  or resolves a feature request, be sure to link to that issue in the
  additional information section.
-->

Update `blebox_uniapi` package version to `2.2.1` (latest). This version incorporates small improvements/fixes to collection of power measurement data of switchBox and switchBoxD devices. Also resolves problem logged regarding power measurements (see #103421).

I'm now official maintainer of `blebox_uniapi` (pinging @bbx-a and @riokuu to confirm) and will be working on this project for some longer time, so I have added myself to blebox component codeowners.

---
**2.2.1**

Changelog:
* fixed support for power measurement capabilities of switchBox and switchBoxD devices

Release: [link](https://github.com/blebox/blebox_uniapi/releases/tag/v2.2.1)
Diff: https://github.com/blebox/blebox_uniapi/compare/v2.2.0...v2.2.1

---

**2.2.2**

Changelog:
* fixed wind reading units to get proper raw m/s value (division by 10, see PR https://github.com/blebox/blebox_uniapi/pull/150)

Release: [link](https://github.com/blebox/blebox_uniapi/releases/tag/v2.2.2)
Diff: https://github.com/blebox/blebox_uniapi/compare/v2.2.1...v2.2.2

---

## Type of change
<!--
  What type of change does your PR introduce to Home Assistant?
  NOTE: Please, check only 1! box!
  If your PR requires multiple boxes to be checked, you'll most likely need to
  split it into multiple PRs. This makes things easier and faster to code review.
-->

- [x] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New integration (thank you!)
- [ ] New feature (which adds functionality to an existing integration)
- [ ] Deprecation (breaking change to happen in the future)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information
<!--
  Details are important, and help maintainers processing your PR.
  Please be sure to fill out additional details, if applicable.
-->

- This PR fixes or closes issue: fixes #103421
- This PR is related to issue: https://github.com/blebox/blebox_uniapi/issues/147 https://github.com/blebox/blebox_uniapi/issues/137
- Link to documentation pull request: n/a

## Checklist
<!--
  Put an `x` in the boxes that apply. You can also fill these out after
  creating the PR. If you're unsure about any of them, don't hesitate to ask.
  We're here to help! This is simply a reminder of what we are going to look
  for before merging your code.
-->

- [x] The code change is tested and works locally.
- [x] Local tests pass. **Your PR cannot be merged unless tests pass**
- [x] There is no commented out code in this PR.
- [x] I have followed the [development checklist][dev-checklist]
- [x] I have followed the [perfect PR recommendations][perfect-pr]
- [x] The code has been formatted using Ruff (`ruff format homeassistant tests`)
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

If the code communicates with devices, web services, or third-party tools:

- [x] The [manifest file][manifest-docs] has all fields filled out correctly.  
      Updated and included derived files by running: `python3 -m script.hassfest`.
- [x] New or updated dependencies have been added to `requirements_all.txt`.  
      Updated by running `python3 -m script.gen_requirements_all`.
- [x] For the updated dependencies - a link to the changelog, or at minimum a diff between library versions is added to the PR description.
- [x] Untested files have been added to `.coveragerc`.

<!--
  This project is very active and we have a high turnover of pull requests.

  Unfortunately, the number of incoming pull requests is higher than what our
  reviewers can review and merge so there is a long backlog of pull requests
  waiting for review. You can help here!
  
  By reviewing another pull request, you will help raise the code quality of
  that pull request and the final review will be faster. This way the general
  pace of pull request reviews will go up and your wait time will go down.
  
  When picking a pull request to review, try to choose one that hasn't yet
  been reviewed.

  Thanks for helping out!
-->

To help with the load of incoming pull requests:

- [ ] I have reviewed two other [open pull requests][prs] in this repository.

[prs]: https://github.com/home-assistant/core/pulls?q=is%3Aopen+is%3Apr+-author%3A%40me+-draft%3Atrue+-label%3Awaiting-for-upstream+sort%3Acreated-desc+review%3Anone+-status%3Afailure

<!--
  Thank you for contributing <3

  Below, some useful links you could explore:
-->
[dev-checklist]: https://developers.home-assistant.io/docs/development_checklist/
[manifest-docs]: https://developers.home-assistant.io/docs/creating_integration_manifest/
[quality-scale]: https://developers.home-assistant.io/docs/integration_quality_scale_index/
[docs-repository]: https://github.com/home-assistant/home-assistant.io
[perfect-pr]: https://developers.home-assistant.io/docs/review-process/#creating-the-perfect-pr

swistakm committed:
Bump blebox_uniapi to 2.2.1 and update blebox component code owners

bbx-a commented:
> Hey there @bbx-a, @riokuu, mind taking a look at this pull request as it has been labeled with an integration (`blebox`) you are listed as a [code owner](https://github.com/home-assistant/core/blob/dev/CODEOWNERS#L160) for? Thanks!

The changes look good!

Welcome to the new code owner @swistakm

edenhaus reviewed (changes requested):
Thanks for bumping the dependency! However, could you please update the PR description to contain at least one (or multiple) of the following:

- A link to the release notes of this package version, and all versions in between.
- A link to the changelog of this package.
- A link to a Git(Hub) diff/compare view from the current version to the bumped version.

This allows us to review upstream changes, which is needed to decide this change is working as intended and/or if we can include it in, for example, a patch release of Home Assistant.

Thanks already! 👍

swistakm committed:
Bump blebox_uniapi to 2.2.2

swistakm commented:
Hey @edenhaus. Sorry for delay. Was busy working on other issues over the week. In the meantime we have fixed another minor issue on the blebox_uniapi side and released another patch release. Here is summary of all the changes in chronological order:

---
**2.2.1**

Changelog:
* fixed support for power measurement capabilities of switchBox and switchBoxD devices

Release: [link](https://github.com/blebox/blebox_uniapi/releases/tag/v2.2.1)
Diff: https://github.com/blebox/blebox_uniapi/compare/v2.2.0...v2.2.1

---

**2.2.2**

Changelog:
* fixed wind reading units to get proper raw m/s value (division by 10, see PR https://github.com/blebox/blebox_uniapi/pull/150)

Release: [link](https://github.com/blebox/blebox_uniapi/releases/tag/v2.2.2)
Diff: https://github.com/blebox/blebox_uniapi/compare/v2.2.1...v2.2.2

---

We have another small fix to be merged soon (https://github.com/blebox/blebox_uniapi/pull/153) but I think it would be better to update in smaller batches. WDYT?

Last but not least, if you by chance want to test evaluate changes locally but don't have any blebox device at hand: I've built a [small simulator](https://github.com/blebox/fakebox) for selected devices.  It is not complete but I plan to extend it over time. It supports zeroconf (but only on macOS) so homeassistant can discover all simulated devices. It makes much easier to evaluate backward compatibility of historical device variants.

swistakm commented:
And the full changelog: https://github.com/blebox/blebox_uniapi/blob/master/HISTORY.rst

swistakm commented:
@edenhaus I have just noticed that automation marked this PR as draft and I completely missed it.

edenhaus reviewed (approved):
Thanks @swistakm :+1: 
Can be merged if CI is passing

edenhaus commented:
> @edenhaus I have just noticed that automation marked this PR as draft and I completely missed it.

No need to ping me as the bot already requested a review from me, when you click "Ready for review"

edenhaus committed:
Merge branch 'dev' into update/bump-blebox-uniapi-version
